### PR TITLE
ref(tsx): use HTMLAttributes on platformpicker

### PR DESCRIPTION
**Pull request by JonasBa**
Before
CleanShot 2022-03-02 at 14 13 57@2x
After
CleanShot 2022-03-02 at 14 14 17@2x

### CODE REVIEW

File: static/app/components/forms/blankField.tsx [hunk 1]
```diff
@@ -1,16 +1,14 @@
 import * as React from 'react';
 
-import Field from 'sentry/components/forms/field';
-
-type Props = Field['props'];
+import Field, {FieldProps} from 'sentry/components/forms/field';
```
Comment (priscilawebdev) [inline static/app/components/forms/blankField.tsx:3]: are these changes related? I'm finding it difficult to review these PRs :/

Comment (JonasBa) [reply static/app/components/forms/blankField.tsx:3]: No, sorry :) needs to be rebased because downstream diverged. Let me do it real quick

Comment (JonasBa) [reply static/app/components/forms/blankField.tsx]: Fixed, sorry about that :)

File: static/app/components/platformPicker.tsx [hunk 2]
```diff
@@ -29,24 +29,24 @@ const PlatformList = styled('div')`
 
 type Category = typeof PLATFORM_CATEGORIES[number]['id'];
 
-type Props = {
+interface PlatformPickerProps {
   setPlatform: (key: PlatformKey | null) => void;
   defaultCategory?: Category;
   listClassName?: string;
-  listProps?: React.ComponentProps<typeof PlatformList>;
+  listProps?: React.HTMLAttributes<HTMLDivElement>;
```
Comment (priscilawebdev) [inline static/app/components/platformPicker.tsx:36]: do we have a PlatformList prop we can use here? is the list a div element?

Comment (JonasBa) [reply static/app/components/platformPicker.tsx:36]: Yes, it's a div element. Idk if it's worth it tbh, I would just not write it for now and when we do need it, we can add it

**Review by priscilawebdev — APPROVED** [review]
